### DISCUSSION TITLE
504 Gateway Time-out on Hubs Cloud instance

### DISCUSSION THREAD

**Discussion by Charlesc22** [mozilla/hubs#3502] [Hubs Cloud Questions] [answered]
Has anyone else seen a "504 Gateway Time-out" error when being in a room after a few minutes? We seem to have a scene that is crashing our server. I have to do a manual update to the stack to take it offline and back online to get it running again.

We have designers working on a specific scene, but something they did to the scene is causing our Hubs cloud server to crash after you are in the room(with scene) after a few minutes.

I'm not even sure how to troubleshoot this. Are there logs on AWS I can look at to see what caused the problem? Or are there any logs on Hubs side that could help detect the problem?

Thanks
Charles

**Comment by Charlesc22** [comment, marked as answer]
We seem to have narrowed the problem down to a 4K(8 hour) video stream from youtube that was in the scene.
When a 2nd person entered the room, the Hubs server would crash and give the 504 Gateway error.
We replaced the video with a 1 minute video we hosted on S3 and that seemed to solve the problem

So not sure if the issue is the size of the video, or the act of synching/networking it with the 2nd instance of the room

**Reply by robinkwilson** [reply]
I've seen some gnarly scenes that download 100+ MB at a time because of videos -- but no 504 timeout problem.

Are you using a streaming service or just including it in the scene? I haven't looked into this myself, but if there's anyway to force it to stream, would be helpful for those very long videos.

**Reply by Charlesc22** [reply]
We were just using the Youtube link to a very long video, so I'm assuming it was streaming it. Not sure if the 4K format could have been the problem?
We switched to a short 2-minute video hosted on AWS S3, and everything was running smoothly again.

**Reply by robinkwilson** [reply]
Wonderful happy to hear! Let me know if there are any other issues!

**Comment by lachlanphillips** [comment]
Adding that I've experienced this as well, also from adding a long youtube video or two.
